flex-checks/src/main/java/org/sonar/flex/checks/DebugFeaturesCheck.java

```diff
@@ -63,14 +64,21 @@
 
   @Override
   public List<AstNodeType> subscribedTo() {
-    return Collections.emptyList();
+    return Collections.singletonList(FlexGrammar.POSTFIX_EXPR);
   }
 
   @Override
   public void visitFile(@Nullable AstNode node) {
     currentState = State.EXPECTING_ALERT;
   }
 
+  @Override
+  public void visitNode(AstNode astNode) {
+    if (isFunctionCall(astNode) && "trace".equals(astNode.getFirstChild().getTokenValue())) {
+      addIssue("Remove this use of the \"trace\" function.", astNode);
```

Comment:
@amelie-renard-sonarsource we should probably update the message to the rspec one, we should also change the line below `addIssue("Remove this usage of Alert.show().", token);`.